src/Engine/ProtoCore/Lang/CallSite.cs

```diff
@@ -1528,6 +1528,27 @@ private FunctionGroup FirstFunctionGroupInInheritanceChain(RuntimeCore runtimeCo
             return ret;
         }
 
+        //Pre-initialize array for repeated calls.  The StackValue is inserted vs making a new array for every call.
+        private static List<StackValue> disposeArguments = new List<StackValue>() { StackValue.Null };
+        //Cache final function endpoint for repeated calls;
+        private FunctionEndPoint finalFep;
+
+        public StackValue DispatchDispose(
+            StackValue stackValue, RuntimeCore runtimeCore)
```

Comment:
Can we have this line on the same line as 1536?

Reply:
Fixed